packages/sidecar/src/overlay.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Spotlight</title>
+    <!-- Set process for development environment -->
+    <script>
+      window.process = { env: { NODE_ENV: 'development' } };
+    </script>
+    <!-- Load Spotlight.js from the CDN -->
+    <script type="module" src="https://unpkg.com/@spotlightjs/overlay@latest/dist/sentry-spotlight.js"></script>
```

Comment:
Yes, this - I think since this is anyway a solution that isn't meant to be the default and it solves the problem, I was like - let's ship it and see what happens.

Reply:
Hrm is there any reason we couldn't just inject the contents of the script? Feels odd to have this load a file thats not the actual file in the dep.

It also requires an internet connection w/ this.

Reply:
I will think of something